src/index.js

```diff
@@ -60,6 +60,16 @@ function generateSafeGetCall(object, property, computed) {
   );
 };
 
+function splitMultipleDirectives(directive) {
+  if (!directive.startsWith('use ')) {
+    return [];
+  }
+
+  return directive.slice(4) // ommit 'use '
+                  .split(', ')
```

Comment:
1. ', ' -> ',' , i.e. don't require the spacing between directives.
2. please correct the formatting. Don't indent the next line more than 1 identation unit (i.e. 2 spaces) more than the previous line.